pocs/focuser/focuser.py

```diff
@@ -12,7 +12,7 @@
 
 from pocs import PanBase
 from pocs.utils import current_time
-from pocs.utils import images
+from pocs.utils.images import focus as focus_utils
```

Comment:
I notice that you always end up renaming it to focus_utils. How about renaming the file to focus_utils.py?

Reply:
I thought that was a little odd since it was already in `utils` folder. I suppose it makes the most sense though.

Reply:
The more I think about this the less I like renaming it. It could potentially be annoying left as is, but I think it's a cleaner name. I'm going to leave as it is now unless there are strong feelings either way.

Reply:
Leave it.